packages/medusa/src/api-v2/admin/regions/route.ts

```diff
@@ -1,20 +1,47 @@
+import { createRegionsWorkflow } from "@medusajs/core-flows"
+import { CreateRegionDTO } from "@medusajs/types"
 import { remoteQueryObjectFromString } from "@medusajs/utils"
 import { MedusaRequest, MedusaResponse } from "../../../types/routing"
-import { defaultAdminRegionFields } from "./query-config"
 
 export const GET = async (req: MedusaRequest, res: MedusaResponse) => {
   const remoteQuery = req.scope.resolve("remoteQuery")
 
-  const variables = { filters: req.filterableFields }
-
   const queryObject = remoteQueryObjectFromString({
     entryPoint: "region",
-    variables,
-    fields: defaultAdminRegionFields,
+    variables: {
```

Comment:
Not necessarily something we need to change in this PR, but wouldn't it make sense for `remoteQueryObjectFromString` to take a `MedusaRequest`, and handle this variable setting internally? Since MedusaRequest is standardized, I don't see any harm in doing that, and it will eliminate this sort of boilerplate.

Eg. the signature will then be `remoteQueryObjectFromString(req, "region")`

Reply:
The idea is that the util can be used by our users as well, and shouldn't rely on express necessarely, that is why we are enforcing the API of the util to accept a specific object instead of knowing about the underlying http server

Reply:
@adrien2p from what I understand this is part of the medusa core package, so customers won't really do changes here? Or is there another use-case for remoteQueryObjectFromString?